.eslintrc.json

```diff
@@ -2,5 +2,6 @@
   "extends": "amex",
   "rules": {
     "unicorn/prevent-abbreviations": "off"
-  }
+  },
+  "ignorePatterns": ["examples/**"]
```

Comment:
Why would we not lint our examples? Examples tend to be copied and pasted a lot. They need to be high quality

Reply:
Create React App and NextJS Sandboxes have their own linting and opinions. Do we wish to enforce our eslint on other ecosystems we demo?